src/app/shared/components/header/components/nav/components/nav-item/nav-item-vm.model.ts

```diff
@@ -0,0 +1,3 @@
+export interface NavItemVM {
+  titleKey: string;
```

Comment:
???

Reply:
Tomorrow, I am going to answer all the questions why I thought it would be better to use this and remove ng-content.